Wrapping/Generators/SwigInterface/CMakeLists.txt

```diff
@@ -388,7 +388,7 @@ macro(itk_end_wrap_module_swig_interface)
           --submodule-order "${WRAPPER_SUBMODULE_ORDER}"
           --pyi_dir "${ITK_STUB_DIR}"
           DEPENDS ${ITK_WRAP_DOC_DOCSTRING_FILES} ${xml_files} ${IGENERATOR} ${typedef_in_files}
-          WORKING_DIRECTORY / # Arguments to WORKING_DIRECTORY may use generator expressions
+          WORKING_DIRECTORY "${ITK_DIR}/Wrapping/$<CONFIG>" # Arguments to WORKING_DIRECTORY may use generator expressions
```

Comment:
It looks like this directory needs to be created beforehand. From failed build: `/bin/sh: line 0: cd: /.../s-build/Wrapping/Release: No such file or directory`